webpack/InsightsCloudSync/Components/InsightsSettings/__tests__/InsightsSettingsSelectors.test.js

```diff
@@ -0,0 +1,21 @@
+import { testSelectorsSnapshotWithFixtures } from '@theforeman/test';
+import {
+  selectSettings,
+  selectInsightsSyncEnabled,
+} from '../InsightsSettingsSelectors';
+import { rhCloudStateWrapper } from '../../../../ForemanRhCloudTestHelpers';
+
+const state = rhCloudStateWrapper({
```

Comment:
travis fails because state helper have changed a bit in #276
can you use `insightsStateWrapper` instead?
```suggestion
const state = insightsStateWrapper({
```